feature-libs/cart/base/components/cart-totals/cart-totals.component.ts

```diff
@@ -1,59 +1,18 @@
-import {
-  ChangeDetectionStrategy,
-  Component,
-  OnDestroy,
-  OnInit,
-} from '@angular/core';
-import {
-  Event,
-  NavigationCancel,
-  NavigationEnd,
-  Router,
-} from '@angular/router';
-import { ActiveCartFacade, Cart, OrderEntry } from '@spartacus/cart/base/root';
-import { Observable, Subscription } from 'rxjs';
-import { filter } from 'rxjs/operators';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { ActiveCartFacade, Cart } from '@spartacus/cart/base/root';
+import { Observable } from 'rxjs';
 
 @Component({
   selector: 'cx-cart-totals',
   templateUrl: './cart-totals.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class CartTotalsComponent implements OnInit, OnDestroy {
+export class CartTotalsComponent implements OnInit {
   cart$: Observable<Cart>;
-  entries$: Observable<OrderEntry[]>;
-  cartValidationInProgress = false;
 
-  protected subscription = new Subscription();
-
-  constructor(
-    protected activeCartService: ActiveCartFacade,
-    protected router?: Router
-  ) {}
+  constructor(protected activeCartService: ActiveCartFacade) {}
```

Comment:
conventions from now for all 'facades', let's name the variable for them activeCartFacade

```suggestion
  constructor(protected activeCartFacade: ActiveCartFacade) {}
```

Reply:
we are mostly using this approach -> activeCartService: ActiveCartFacade, but also agree that in the repo there are 2 ways.